packages/ui/src/components/ConnectCreateOrWatch.tsx

```diff
@@ -0,0 +1,69 @@
+import { styled } from '@mui/material'
+import { useAccounts } from '../contexts/AccountsContext'
+import { useNavigate } from 'react-router-dom'
+import { Button } from './library'
+import { WATCH_ACCOUNT_ANCHOR } from '../pages/Settings/Settings'
+
+export const ConnectOrWatch = () => {
+  const { isAllowedToConnectToExtension, allowConnectionToExtension } = useAccounts()
+  const navigate = useNavigate()
+
+  return (
+    <WrapperStyled>
+      {isAllowedToConnectToExtension ? (
+        <div>No multisig found for your accounts or watched accounts.</div>
+      ) : (
+        <>
+          <h1>Multix is an interface to easily manage complex multisigs.</h1>
+          <div>Connect a wallet to interact with Multix or watch an address.</div>
```

Comment:
```suggestion
          <div>Connect a wallet to interact with Multix or watch an account.</div>
```

We should update this too now the button name has changed

Reply:
Makes sense, thanks!